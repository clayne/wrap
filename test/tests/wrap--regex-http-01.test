wrap | /dev/null | -w50 | regex-http-01.txt | 0
