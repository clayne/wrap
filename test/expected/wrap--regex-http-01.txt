This is an example HTTP URI:
http://www.example-site.com
